fdbclient/include/fdbclient/CommitProxyInterface.h

```diff
@@ -492,6 +495,7 @@ struct GetBlobGranuleLocationsRequest {
 	Optional<KeyRef> end;
 	int limit;
 	bool reverse;
+	bool justGranules;
```

Comment:
After reading the PR description and the code changes, it's still not obvious to me as to what distinction this parameter makes. Comment here could help. Also, does it make sense to have separate request streams?

Reply:
Yes the request is identical apart from this one parameter, and the server code is very similar. I'll add a comment